packages/storybook/src/schematics/component-story/index.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  chain,
+  externalSchematic,
+  Rule,
+  Tree,
+  SchematicContext,
+  mergeWith,
+  apply,
+  url,
+  template,
+  move
+} from '@angular-devkit/schematics';
+import { PropertyDeclaration, SyntaxKind } from 'typescript';
+import {
+  getSourceNodes,
+  findNodes
+} from '@schematics/angular/utility/ast-utils';
```

Comment:
Let's not depend on `schematics/angular` since this should support React and web components as well.